PWGJE/Tasks/hadronPhotonCorrelation.cxx

```diff
@@ -462,27 +451,52 @@ struct HadronPhotonCorrelation {
   PROCESS_SWITCH(HadronPhotonCorrelation, processPhotonCorrelationsMCReco, "hadron-photon correlation", true);
 
   void processPhotonCorrelationsMCGen(JetMcCollision const&,
-                                      JetParticles const& tracks_true)
+                                      Join<JetParticles, JMcParticlePIs> const& tracks_true)
```

Comment:
why are the PI tables added here? do you need the original particle table for some reason?